tests/pytorch/attention/test_attention.py

```diff
@@ -1025,13 +1052,18 @@ def get_dummy_cuda_rng_tracker() -> CudaRNGStatesTracker:
         fast_zero_fill=True,
     )
     if is_training:
-        out.backward(d_out)
+        out.backward((d_out, torch.zeros(1,device="cuda")))
+
+    if config.return_max_score:
+        out = (out, max_score)
+    else:
+        out = (out, None)
```

Comment:
This changes returned values when max score is not requested. Should we keep backward compatibility if new feature is not used?